turbo.json

```diff
@@ -1,13 +1,19 @@
 {
   "$schema": "https://turbo.build/schema.json",
   "tasks": {
+    "ready": {
+      "dependsOn": ["^ready"],
```

Comment:
why ready depending on ready?

Reply:
[d83753b](https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/503/commits/d83753bc5950fdf75167d4ff07909673ce746379)

mistake... fixed it!